napari_animation/interpolation.py

```diff
@@ -24,13 +24,18 @@ def default(a, b, fraction):
     ----------
         Interpolated value between a and b at fraction.
     """
-    if isinstance(a, numbers.Number) and isinstance(b, numbers.Number):
+    if isinstance(a, bool) or isinstance(b, bool):
+        # checking this first because booleans are numbers
+        return interpolate_bool(a, b, fraction)
+
+    elif isinstance(a, numbers.Number) and isinstance(b, numbers.Number):
         return interpolate_num(a, b, fraction)
 
     elif isinstance(a, (list, tuple)) and isinstance(b, (list, tuple)):
         return interpolate_seq(a, b, fraction)
 
     else:
+        # strings, etc.
         return interpolate_bool(a, b, fraction)
```

Comment:
do we know what actually gets caught in here? do we always want the instant transition at t0? I think we do but just trying to work out if this will lead to unexpected behaviour anywhere

Reply:
Changes in opacity, translation, scale (? never tried it), are caught by the `interpolate_num` or the `seq` and later with `num`.

Changes in colormap and properties (? never tried neither) are caught in `else`. The issue is that they are not well defined with other interpolation operations. Changing them instantly I think covers most of the applications.

Reply:
ideal - I think this is the expected behaviour then! :)